app/services/rapid_ready_for_decision/processor_selector.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module RapidReadyForDecision
+  class ProcessorSelector
+    def initialize(form526_submission)
+      @form526_submission = form526_submission
+    end
+
+    def processor_class
+      return nil unless rrd_enabled?
+
+      if hypertension_enabled? && single_issue_claim_applicable?(DiagnosticCodes::HYPERTENSION)
+        return RapidReadyForDecision::DisabilityCompensationJob
+      end
+
+      nil
+    end
+
+    def rrd_applicable?
+      !processor_class.nil?
+    end
+
+    def self.disability_increase?(disability, diagnostic_code)
+      disability['diagnosticCode'] == diagnostic_code &&
+        disability['disabilityActionType'] == 'INCREASE'
+    end
+
+    private
+
+    def rrd_enabled?
+      # In next PR, change this to be a Flipper configuration to disable RRD completely
+      true
+    end
+
+    def hypertension_enabled?
```

Comment:
It could be premature to make this method more generalized and applicable to any contention's Flipper check, but the upside is that it might enforce some kind of naming standard for the Flipper feature. Just a thought!

Reply:
Are you suggesting a change that looks something like this:
```suggestion
    def hypertension_enabled?(disability)
      Flipper.enabled?("disability_#{disability}_compensation_fast_track".to_sym)
```
If so, I like it. We may want to rename the Flipper key to include `rrd` or something to that effect.

Reply:
Yeah, that looks like what I was thinking of! Maybe the name could be `rrd_#{disability}_compensation` or something like that?